build/cqs.tsv

```diff
@@ -170,6 +181,7 @@ org.scalatra:scalatra-swagger_2.11	2.3.0	compile
 org.slf4j:slf4j-api	1.7.5	compile
 org.slf4j:slf4j-log4j12	1.7.5	compile
 org.tukaani:xz	1.0	compile
+org.xerial:sqlite-jdbc	3.8.11.2	compile
```

Comment:
No, actually it's a cqengine dependency. I think cqengine uses it for parsing SQL.